<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>481</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameTextBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTextBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableMessages">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Message ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Message Subject</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="messagesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Messages NOT to show alerts on:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRemoveSelected">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>100</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove selected</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showPasswordCheckBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show password</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="startWithWindowsCheckBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start with Windows</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemoveSelected</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>removeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showPasswordCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>OptionsDialog</receiver>
   <slot>togglePassword(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeSelected()</slot>
  <slot>togglePassword(int)</slot>
 </slots>
</ui>
